src/elm/Pages/Build/Model.elm

```diff
@@ -40,6 +40,16 @@ type alias PartialModel a =
     }
 
 
+{-| BuildModel : model to contain build information that is crucial for rendering a pipeline
```

Comment:
adding an argument model so we dont have to pass 12 arguments to every single Build.View function